test/io.jl

```diff
@@ -379,6 +379,14 @@ module TestIO
     @test_throws DimensionMismatch writetable(tf, df3b, header = false, append = true)
     rm(tf)
 
+    # create a new file if one does not exist. then, append to it.
```

Comment:
"if one does not exist" sounds weird, since you specifically remove it before creation.